src/libs/tools/darktable.c

```diff
@@ -48,6 +48,7 @@ typedef struct dt_lib_darktable_t
   int image_width, image_height;
   // text with logo font
   cairo_surface_t *text;
+  guint8 *text_img_buffer;
```

Comment:
can you comment on this change, I don't get it.
- you create a new variable
- set t
- free it

But never use it ?

Reply:
it's for text logo svg buffer, which leaked memory... i can make it a bit better without need for keeping buffer :)